addons/friendly_tracker/initSettings.sqf

```diff
@@ -0,0 +1,58 @@
+[
+	QGVAR(enabled), // Global var name
+	"CHECKBOX", // Setting type
+	[LSTRING(Enabled), LSTRING(Enabled_Description)], // Title, Tooltip
+	LSTRING(DisplayName),
+	true, // Extra params, depending on settings type
+	true, // Is global
+	{
+		if (_this) exitWith {
+			[] call FUNC(init);
+		};
+	}, // Init/Stop script
+	false // Needs mission restart
+] call CBA_Settings_fnc_init;
```

Comment:
`CBA_Settings_fnc_init` is deprecated, it's recommended to use `CBA_fnc_addSetting` which is safer and does not require to have `cba_settings` in required addons.

Reply:
Good catch. I think our mission preset settings also need this upgrade.